src/base/net/portforwarder.h

```diff
@@ -64,8 +74,11 @@ namespace Net
 
         bool m_active;
         libtorrent::session *m_provider;
+#if LIBTORRENT_VERSION_NUM < 10200
         QHash<quint16, int> m_mappedPorts;
-
+#else
+        QHash<quint16, std::vector<libtorrent::port_mapping_t>> m_mappedPorts;
```

Comment:
the change to support multiple mappings per port seems orthogonal to the 1.2 update, no?

Reply:
@zeule, please don't introduce any non required changes in this PR.

Reply:
I remember I was making only compilation fixes in the commits used as sources for this PR, but let me check.